.github/workflows/ci.yml

```
@@ -21,6 +21,18 @@ jobs:
             - name: Run jest
               run: yarn testJest
 
+    eslint:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Clone repository
+              uses: actions/checkout@v3
+
+            - uses: ./.github/actions/setup-node-yarn-deps
+
+            - name: Ru eslint
```

another day, another typo 😄 
```suggestion
            - name: Run eslint
```